spec/features/admin/teachers/reopen_induction_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe "Admins can reopen a teacher's closed induction" do
+  let(:teacher) { FactoryBot.create(:teacher) }
+  let!(:induction_period) do
```

Comment:
You might have noticed I'm in the habit of favouring the before block; unless something needs to be accessed in an assertion, I don't name a variable.

Reply:
Tbh I loathe `let!` anyway, I'm just trying to fit in 😆